enhancements/security/runtime-class-scc.md

```diff
@@ -0,0 +1,248 @@
+# openshift scc runtime class enhancement
+---
+title: gate-runtime-classes-with-scc
+authors:
+  - "@haircommander"
+  - "@mrunalp"
+reviewers:
+  - "@deads2k"
+  - "@sttts"
+approvers:
+  - ""
+creation-date: 2020/12/04
+last-updated: 2020/12/11
+status: provisional
+
+---
+# Gate Runtime Classes with SCC
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Enhance the Openshift SCC with two fields called `AllowRuntimeClasses` and `EnsureRuntimeClasses`, which gate users from specifying runtime classes in their Pod requests. 
+
+## Motivation
+
+Enhancing SCC to be runtime class aware allows admins to customize how their users are able to access different runtime customizations. This is key in enabling new features like unprivileged builds, kata containers, and performance enhancements for latency sensitive workloads, while also providing protection from less privileged users gaining access to features they should not.
```

Comment:
thoughout: please break on new sentences or 120ish characters, whichever is shorter.  This allows tighter commenting and still renders correctly.